### PR TITLE
i18n: a2. home, loading, checkurlbanner, basic notifications

### PR DESCRIPTION
**Pull request by biwano**
## Description

Added translation tags for home, loading, checkurlbanner, basic notifications components.

## Related Ticket

https://github.com/KlimaDAO/klimadao/issues/157

## Checklist

- [X] Building the site with `npm run build-site` works without errors
- [X] Building the app with `npm run build-app` works without errors
- [X] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

**Comment by biwano** [comment]
The loading component never shows on my computer, is it still needed?

**Review by Atmosfearful — APPROVED** [review]
@biwano yup we need `<Loading />` as a hacky fix to make react-router's client-side `HashRouter` compatible with server-side next.js. Next really isn't a good choice for SPAs like this but I am stubborn and want both apps to have as much overlap as possible :)

File: app/components/CheckURLBanner/index.tsx [hunk 1]
```diff
@@ -22,18 +23,23 @@ export const CheckURLBanner: FC<Props> = ({ onHide }) => {
...
-            ⚠️ Verify the URL and bookmark this page!
+            <Trans id="checkurlbanner.verify_url_and_bookmark">
+              ⚠️ Verify the URL and bookmark this page!
+            </Trans>
           </Text>
           <Text t="caption" color="lighter" align="center">
-            <strong>dapp.klimadao.finance</strong> is the only official domain.
+            <strong>dapp.klimadao.finance</strong>{" "}
+            <Trans id="checkurlbanner.is_the_only_official_domain">
+              is the only official domain.
```
Comment (Atmosfearful) [inline app/components/CheckURLBanner/index.tsx:33]: I don't think all translations will follow the format of `<noun> is`
so we should wrap the whole thing in a tag right?

**Review by Atmosfearful — CHANGES_REQUESTED** [review]

File: app/components/CheckURLBanner/index.tsx (hunk 1 above)
Comment (biwano) [reply app/components/CheckURLBanner/index.tsx]: You are right.
I also put a comment to help translators

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**